### DISCUSSION TITLE
FB&D - Rotten Dreams v4.9 - 1.21.40+ Support

### DISCUSSION THREAD

**Discussion by Hirxs** [Hirxs-MC/FD-Bedrock-development#188] [Announcements]
![Image](https://hirxs-website.net/wp-content/uploads/2024/10/Screenshot-2024-10-22-191731.png)

# I guess you already know why the add-on was updated today, right?

## SUPPORT
- 1.21.30 > **1.21.40/1**

## CHANGES
- The textures of the "party flags" block have been redesigned, including all its variants
- All party tables were redesigned
- Wall banners (red and purple) have been redesigned
- The "wall structure" block has been slightly redesigned

## FIXES
- Fixed a bug where the hitbox of the "Party flags" block had the same hitbox as the "cupcake" block

#### SUPPORT
- 1.21.30 > **1.21.40/1**
#### SUPPORT
- 1.21.30 > **1.21.40/1**

<hr /><em>This discussion was created from the release <a href='https://github.com/Hirxs-Workshop/FD-Bedrock-development/releases/tag/official-release-v4.9'>FB&D - Rotten Dreams v4.9 - 1.21.40+ Support</a>.</em>

**Comment by mithy-mc** [comment]
Hi, the update is very good! The only problem is that the advanced walls are not working, I hope you will get an upcoming update, as this feature is great!

**Reply by Hirxs** [reply]
Greetings, advanced walls do work in 1.21.40+, try removing the pack from MC settings in the "storage" section and re-import it

**Reply by mithy-mc** [reply]
It worked! thank you

**Comment by mithy-mc** [comment]
Hello again, I came to give you an idea, I was going to talk about you putting new arcades and adding new things from the movie like poster, objects and more, it's just an idea, thank you!
